cirq-core/cirq/devices/fidelity.py

```diff
@@ -0,0 +1,216 @@
+from typing import Sequence
+from itertools import product
+import cirq
```

Comment:
Cirq's import hierarchy means using `import cirq` in non-test code may lead to issues:
https://github.com/quantumlib/Cirq/blob/782efcd04c3bbf73a0d630306a3d1cfd9966521d/cirq-core/cirq/__init__.py#L17-L20

Instead, only import the package that this file needs (compare with [noise_model.py](https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/devices/noise_model.py)). Type annotations which aren't explicitly used in this file can be quoted (e.g. `NoiseModelFromFidelity('cirq.NoiseModel')`); to keep the type checker happy, you may need also to copy this from the top of `noise_model.py`:
```
if TYPE_CHECKING:
    from typing import Iterable
    import cirq
```